nemo/core/neural_modules.py

```diff
@@ -393,6 +486,7 @@ def num_weights(self):
         pass
 
     @staticmethod
```

Comment:
this is 100% not working now, right? So lets remove

Reply:
well, it is extracting _something_ ... my problem is that I have never really understood what you wanted to store down there... or why...